src/manipulation.js

```diff
@@ -17,7 +17,7 @@ define([
 
 var
 	rxhtmlTag = /<(?!area|br|col|embed|hr|img|input|link|meta|param)(([\w:]+)[^>]*)\/>/gi,
-	rtagName = /<([\w:]+)/,
+	rtagName = /<([\w:-]+)/,
```

Comment:
> But what we should definitely do is add tests for these cases.

I'd appreciate if you could be explicit about which tests cases (nodes) you still want to include. I'm glad to add as many as required to ensure the change is well covered.